src/Open3D/Visualization/Visualizer/GuiVisualizer.cpp

```diff
@@ -186,6 +179,19 @@ class DrawTimeLabel : public gui::Label {
     gui::Window *window_;
 };
 
+class SmallButton : public gui::Button {
+    using Super = Button;
+
+public:
+    SmallButton(const char *title) : Button(title) {}
```

Comment:
Could you make this constructor `explicit`?

Reply:
Good catch!

Reply:
Thanks to Codacy))